src/analyzer/analyzersilence.cpp

```diff
@@ -9,18 +9,31 @@ constexpr float kSilenceThreshold = 0.001;
 // TODO: Change the above line to:
 //constexpr float kSilenceThreshold = db2ratio(-60.0f);
 
-}  // anonymous namespace
+bool shouldUpdateMainCue(CuePosition cue) {
```

Comment:
The function makes only sense if we pass a main cue, right?

So I think we should rename
cue -> mainCue